test/unit/advertisement-operator/config-watcher_test.go

```diff
@@ -127,6 +162,34 @@ func testDisableBroadcaster(t *testing.T) {
 	assert.Equal(t, advtypes.AdvertisementDeleting, adv2.Status.AdvertisementStatus)
 }
 
+func waitEventOnClusterConfig(client *crdClient.CRDClient, resourcetype string, name string) error {
```

Comment:
since this is also used for Advertisement I would simply rename it to "waitEvent" or something similar